### DISCUSSION TITLE
Using Gnome Boxes as Quickemu GUI

### DISCUSSION THREAD

**Discussion by sslaia** [quickemu-project/quickemu#1472] [Ideas]
I am somehow not a huge fan of Quickgui. So today I tried to access the quickemu disk with Gnome Boxes. I did expect some issues, but hey, it actually works without issues and without extra configuration. I just imported the existing vm disk by clicking on **Install from file** in Boxes **Create a new virtual machine** and selecting the existing quickemu disk. That's it! Doing the same with virt manager would require additional configuration.

This is a simple solution for people who want to use a GUI for Quickemu vms. A bonus: copy and paste between host and guest works.

**Comment by lj3954** [comment]
Keep in mind that GNOME Boxes has its own configuration, and won't respect quickemu configuration, or have the same compatibility.

**Reply by sslaia** [reply]
That's correct. The sh file in the quickemu vm directory will be not touched. Gnome Boxes keeps its configuration file in its default directory.
